examples/prometheus/rules/kube.rules

```diff
@@ -0,0 +1,55 @@
+groups:
+- name: kubernetes-rules
+  rules:
+
+    - alert: DockerLatencyHigh
+      expr: max(kubelet_docker_operations_latency_microseconds{quantile="0.9"}) / 1e+06 > 1
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: Docker latency is high
+        description: "Docker latency is {{ $value }} seconds for 90% of kubelet operations"
+        alertType: latency
+        component: container runtime
+        selfHealing: false
+        url:
+
+    - alert: KubernetesAPIDown
```

Comment:
Usually you create an additional rule that fires when Prometheus hasn't discovered any API server (eg the service discovery is broken).

```
- alert: KubernetesAPIAbsent
     expr: absent(up{job="kubernetes-apiservers"})
     for: 10m
     ...
```